### ISSUE TITLE
 show facets onclick?

### ISSUE THREAD

**Issue by danielebailo** [documentcloud/visualsearch#45]
This post can be found also here http://stackoverflow.com/questions/9432478/visualsearch-js-how-to-show-facets-onclick

A feature which, according to my users feedback, is realy missing, is the possiblity to know, somehow, which are the available facets. My facets are many and it would be really nice to have a list of facets (maybe grouped) when clicking on the searchbox.

I had a look on the annotated source code, in particular to search_parser.js and to search_box.js.

I suppose that I've to trigger the click event on the element with VS-search-inner id. Or maybe another way would be to work on the parse function... I'm a little bit confused.

I also noticed the showFacetCategoryMenu which, unused at the moment, might implement the feature I'm searching for if modifyed.

So, in conclusion, my question are:

can anybody put me in the right direction?
which would be the right place to put my hands on, in order to implement the "show available facets" function?
